### PR TITLE
Update config node

### PR DESCRIPTION
**Pull request by arxeiss**
# Checklist

- [x] `make test` passes
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [x] commit message follows [commit guidelines](./doc/guides/commit-message.md#commit-message-guidelines)

## Description of change

- Updated SDK functions for missing types
- Fixed `fmt` and `goimports` changed to `gci`

### CODE REVIEW

**Review by bakkelim — APPROVED** [review]

File: grpc/credentials_test.go [hunk 1]
```diff
@@ -52,7 +50,7 @@ var _ = Describe("Test JSON and YAML to JSON credentials", func() {
 		func(filePath string) {
 			// Prepare test data
 			// #nosec: G304: Potential file inclusion via variable
-			data, err := ioutil.ReadFile(filePath)
+			data, err := os.ReadFile(filePath)
```
Comment (maaland) [inline grpc/credentials_test.go:53]: Why this change?

**Review by maaland — APPROVED** [review]

File: grpc/credentials_test.go (hunk 1 above)
Comment (arxeiss) [reply grpc/credentials_test.go:53]: `ioutil` is deprecated. Linter was complaining about it
